easybuild/easyblocks/g/gurobi.py

```diff
@@ -0,0 +1,45 @@
+import os
+import shutil
+
+from easybuild.easyblocks.generic.tarball import Tarball
+from easybuild.tools.build_log import EasyBuildError
+
+
+class EB_Gurobi(Tarball):
+    """Support for installing linux64 version of Gurobi."""
+
+    def configure_step(self):
+        """No configuration for Gurobi."""
+        # ensure a license file is specified
+        if self.cfg['license_file'] is None:
+            raise EasyBuildError("No license file specified.")
+
+    def install_step(self):
+        """Install Gurobi and license file."""
+
+        super(EB_Gurobi, self).install_step()
+
+        # copy license file
+        lic_path = os.path.join(self.installdir, 'gurobi.lic')
+        try:
+            shutil.copy2(self.cfg['license_file'], lic_path)
+        except OSError, err:
```

Comment:
style nitpicking (slowly looking forward to making things Py3 compatible): please use `except OSError as err`